### ISSUE TITLE
UFCU Sync errors

### ISSUE THREAD

**Issue by jw6jw6jw6** [JohnStarich/sage#30]
UFCU Sync is encountering an error

> Failed downloading transactions: [University Federal Credit Union]: Error running action *chromedp.Selector with index #1: &{sel:a[aria-label="History Export"] exp:1 by:0xd9b860 wait:0xd9b4d0 after:<nil> raw:false}: context deadline exceeded
Failed downloading transactions: [University Federal Credit Union]: Error running action *chromedp.Selector with index #1: &{sel:a[aria-label="History Export"] exp:1 by:0xd9b860 wait:0xd9b4d0 after:<nil> raw:false}: context deadline exceeded

**Comment by jw6jw6jw6** [comment]
@JohnStarich I am receiving the same error on Ally

> Failed downloading transactions: [Ally]: Failed to prepare statement download request: Error running action *chromedp.Selector with index #2: &{sel:#accounts-menu-item exp:1 by:0xd9ba90 wait:0xd9b4d0 after:0xd9c620 raw:false}: context deadline exceeded

**Comment by JohnStarich** [comment]
Still working on this one, started on Ally Bank. I need to get creative though, Ally remember which IPs I've logged in with (I think) so it no longer prompts me for 2FA at my home or cell IP.

I also purchased access to a VPN but Ally blocks all VPN locations I've tried... Hoping I don't have to drive to random coffee shops in the middle of quantize just to get new IPs.

Progress so far will be merged here: https://github.com/JohnStarich/sage/pull/31
Will post back as progress is made.
